terraform/modules/prom-ec2/prometheus/cloud.conf

```
@@ -16,17 +16,17 @@ write_files:
         @reboot root mount /dev/nvme0 /mnt
         @reboot root /root/watch_prometheus_dir
   - owner: root:root
-    path: /etc/cron.d/targets_pull
+    path: /etc/cron.d/ireland_targets_pull
     permissions: 0755
     content: |
         # if targets bucket exists then sync it, otherwise this cron runs but has no effect
-        * * * * * root [ "${targets_bucket}" != "" ] && aws s3 sync s3://${targets_bucket}/active/ /etc/prometheus/targets --region=${region} --delete
+        * * * * * root [ "${ireland_targets_bucket}" != "" ] && aws s3 sync s3://${ireland_targets_bucket}/active/ /etc/prometheus/ireland_targets --region=${region} --delete
```

```suggestion
        * * * * * root [ "${ireland_targets_bucket}" != "" ] && aws s3 sync s3://${ireland_targets_bucket}/active/ /etc/prometheus/ireland-targets --region=${region} --delete
```
kebab case for directories please (and for consistency with your prometheus config)